components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1745,6 +1745,16 @@ where
         let stepped = Cell::new(false);
         let memtrace_raft_entries = &mut self.fsm.peer.memtrace_raft_entries as *mut usize;
         defer!({
+            let do_check = || {
+                fail_point!("memtrace_raft_messages_overflow_check_peer_recv", |_| {
+                    true
+                });
+                false
+            };
+            if do_check() {
+                let prev = MEMTRACE_RAFT_MESSAGES.sum();
+                assert!(prev >= heap_size);
```

Comment:
Do you mean:
```rust
fail_point!(
    "memtrace_raft_messages_overflow_check_peer_recv",
    MEMTRACE_RAFT_MESSAGES.sum() < heap_size,
    |_| {}
);

// in test
fail::cfg("memtrace_raft_messages_overflow_check_peer_recv", "panic").unwrap();
```
?

Reply:
Correct

Reply:
Why not adopt the above suggestion, I think it is cleaner

Reply:
Done!